### ISSUE TITLE
Widget value tooltips ignore numericGroupSeparator/numericGroupSize/decimalSeparator

### ISSUE THREAD

**Issue by eduard93** [intersystems-community/DeepSeeWeb#228]
Widget value tooltips should respect numericGroupSeparator/numericGroupSize/decimalSeparator MDX properties.

[Dash](http://161.97.80.143:52777/dsw/index.html#/MAP/map/Map.dashboard)

Widget `Бублик` has the following MDX:

```json
{ 
    "MDX": "SELECT %LABEL([crimeRegionD].[Total],\"Правонарушений\",\"\") ON 0,{[Measures].[crimeWeight3],[Measures].[crimeWeight4],[Measures].[crimeWeight5],[Measures].[crimeWeight7],[Measures].[crimeWeight21]} ON 1 FROM [CRIME]"
}
```

<details>
  <summary>Returning this json</summary>

```json
{
    "Cols": [
        {
            "tuples": [
                {
                    "caption": "Правонарушений",
                    "cellStyle": "",
                    "dimension": "",
                    "format": "",
                    "headerStyle": "",
                    "path": "",
                    "title": "",
                    "total": "",
                    "type": "mbr",
                    "valueID": "",
                    "vis": 1
                }
            ]
        },
        {
            "tuples": [
                {
                    "caption": "Тяжких злочинів",
                    "cellStyle": "",
                    "dimension": "Тяжких злочинів",
                    "format": "",
                    "headerStyle": "",
                    "path": "",
                    "title": "",
                    "total": "SUM",
                    "type": "msr",
                    "valueID": "",
                    "vis": 1
                },
                {
                    "caption": "Особливо тяжких злочинів",
                    "cellStyle": "",
                    "dimension": "Особливо тяжких злочинів",
                    "format": "",
                    "headerStyle": "",
                    "path": "",
                    "title": "",
                    "total": "SUM",
                    "type": "msr",
                    "valueID": "",
                    "vis": 1
                },
                {
                    "caption": "Кримінальних проступків",
                    "cellStyle": "",
                    "dimension": "Кримінальних проступків",
                    "format": "",
                    "headerStyle": "",
                    "path": "",
                    "title": "",
                    "total": "SUM",
                    "type": "msr",
                    "valueID": "",
                    "vis": 1
                },
                {
                    "caption": "Нетяжких злочинів",
                    "cellStyle": "",
                    "dimension": "Нетяжких злочинів",
                    "format": "",
                    "headerStyle": "",
                    "path": "",
                    "title": "",
                    "total": "SUM",
                    "type": "msr",
                    "valueID": "",
                    "vis": 1
                },
                {
                    "caption": "Правопорушення середньої та невеликої тяжкості",
                    "cellStyle": "",
                    "dimension": "Правопорушення середньої та невеликої тяжкості",
                    "format": "",
                    "headerStyle": "",
                    "path": "",
                    "title": "",
                    "total": "SUM",
                    "type": "msr",
                    "valueID": "",
                    "vis": 1
                }
            ]
        }
    ],
    "Data": [
        134649,
        20355,
        126534,
        54892,
        96732
    ],
    "Info": {
        "colCount": 5,
        "colKey": "ru2428061728",
        "cubeClass": "map.cubes.Crime",
        "cubeKey": "CRIME",
        "cubeName": "Crime",
        "decimalSeparator": ",",
        "numericGroupSeparator": " ",
        "numericGroupSize": 3,
        "percentDone": 100,
        "queryKey": "ru2049623924",
        "rowCount": 1,
        "rowKey": "ru3797325569"
    }
}
```
</details>

These properties from the response:

```json
{
  "Info": {
        "decimalSeparator": ",",
        "numericGroupSeparator": " ",
        "numericGroupSize": 3,
  }
}
```
 are currently ignored for tooltips

![image](https://user-images.githubusercontent.com/5127457/142606063-b8bb6b46-18e2-495c-95ba-de84381ce631.png)

Same on other widget types: 

![image](https://user-images.githubusercontent.com/5127457/142606205-9fce60c2-435d-49f7-8543-cea40d43f9d3.png)

**Comment by gnibeda** [comment]
fixed in 3.1.35

**Comment by gnibeda** [comment]
implemented in 3.1.35
* added `percentageFormat` data property, to set format of percents for widgets that show percentage values, e.g. `#.##`

**Comment by eduard93** [comment]
Not fixed in treemap
![image](https://user-images.githubusercontent.com/5127457/143841750-f6ab9f73-241b-4908-8c6f-66cb7b4ced67.png)

**Comment by gnibeda** [comment]
fixed in 3.1.38
